lib/tasks/report_inconsistent_facet_values.rake

```diff
@@ -0,0 +1,57 @@
+require "gds_api/rummager"
+require "json"
+
+class EmptyFacetValueError < StandardError
```

Comment:
An "empty facet" is perhaps more the symptom than the cause. Maybe call this something like `DataInconsistencyError`?